app/error-handler.js

```diff
@@ -9,22 +9,22 @@
     // Fallback for all uncought errors
     function handleError (event, err) {
         try {
-            var msg = document.getElementById('noVNC_fallback_errormsg');
+            const msg = document.getElementById('noVNC_fallback_errormsg');
```

Comment:
This file isn't transpiled when using things directly, so it cannot use any constructs that don't work on all browsers.

Reply:
Do you mean the whole app folder?